src/components/dls/QuranWord/QuranWord.tsx

```diff
@@ -1,60 +1,28 @@
 import React from 'react';
-import pad from 'lodash/pad';
-import WordWrap from './WordWrap';
-import WordGlyph from './WordGlyph';
-import WordType from '../../../../types/WordType';
-import { WORD_TYPES } from '../../../constants/words';
-import WordTransparent from './WordTransparent';
+import WordType from 'types/WordType';
+import IndoPakWordText from './IndoPakWordText';
+import MadaniWordText from './MadaniWordText';
+import UthmaniWordText from './UthmaniWordText';
 
 type QuranWordProps = {
   word: WordType;
-  setCurrentWord?: $TsFixMe;
-  pause?: $TsFixMe;
-  setCurrentVerseKey?: $TsFixMe;
-  playCurrentWord?: $TsFixMe;
-  tooltip: 'translation' | 'transliteration';
-  audioPosition?: number;
-  isCurrentVersePlaying?: boolean;
-  isSearched?: boolean;
-  useTextFont?: boolean;
+  fontStyle?: 'madani' | 'uthmani' | 'indopak';
+  highlight?: boolean;
 };
 
-const QuranWord = ({
-  word,
-  isCurrentVersePlaying = false,
-  audioPosition,
-  useTextFont,
-}: QuranWordProps) => {
-  let text = '';
+const QuranWord = (props: QuranWordProps) => {
+  const { word, fontStyle } = props;
+  let WordText;
 
-  // const className = `${useTextFont ? 'text-' : ''}${word.className} ${word.charType} ${
-  //   word.highlight ? word.highlight : ''
-  // }`;
-
-  if (useTextFont) {
-    if (word.charType === WORD_TYPES.CHAR_TYPE_END) {
-      text = pad(word.verseKey.split(':')[1], 3, '0');
-    } else if (word.textMadani) {
-      text = word.textMadani;
-    }
+  if (fontStyle === 'uthmani') {
```

Comment:
why not do `if` `else` with return? 

```typescript
if (fontStyle === 'uthmani') {
  return <UthmaniWordText code={word.code} pageNumber={word.pageNumber} />;
}
```

Reply:
In the future, the return value will be: 
```
<WordWrapper ...>
	<WordText />
<WordWrapper>
```

WordWrapper will handle the shared functionality across different scripts such as highlighting a word, onClick, etc. 

Lmk if it makes sense!

Reply:
ok